<?xml version="1.0" ?>
<launch>
    <node name="path_follower" pkg="pmadmu_planner" type="path_follower.py" output="screen">
        <param name="robot_id" type="int" value="1"/>
    </node>
</launch>